tmt/steps/prepare/ansible.py

```diff
@@ -1,7 +1,25 @@
+import sys
+from typing import Any, List, Optional
+
 import click
 
 import tmt
 
+if sys.version_info >= (3, 8):
+    from typing import TypedDict
+else:
+    from typing_extensions import TypedDict
+
+# Type annotation for "data" package describing a guest instance. Passed
```

Comment:
@guy9050 this comment seems a copy-paste, this is not at all describing a guest instance, more like Ansible prepare step?

Reply:
I think yes. I'll remove it.

Reply:
This does not seem to be resolved, the `guest instance` comment is still present.

Reply:
Thanks.